pkg/util/pki/csr_test.go

```diff
@@ -369,3 +371,91 @@ func TestRemoveDuplicates(t *testing.T) {
 		}
 	}
 }
+
+func TestGenerateCSR(t *testing.T) {
+	asn1KeyUsage, err := asn1.Marshal(asn1.BitString{Bytes: []byte{0xa0}, BitLength: asn1BitLength([]byte{0xa0})})
```

Comment:
I guess these are the DigitalSignature and Encipherment usages?
Maybe add a comment or use the named variables  from the keyusages.go module?

Reply:
I didn't use the names variables as it would create 10 more lines doing binary logic, added a note